DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -725,12 +800,14 @@ function PreFirstEntryStartMapFixes(#var(PlayerPawn) player, FlagBase flagbase,
 
         case 129:
             MarkConvPlayed("GaryHostageBriefing", bFemale);
+            MarkConvPlayed("DL_no_carla", bFemale); // Majestic 12 is attacking Vandenberg... in 120+ starts, you don't need fing the keypads
```

Comment:
but you might want to release the bots, this gives you the code

I don't think Jock says any incorrect info here with later starts?

Reply:
or maybe these later starts should cause the power to be restored already so you can easily release the bots?

Reply:
My comment was wrong. The problem is "Base is under lockdown until the four MJ12 bots can be destroyed". The base is not under lockdown in 121+ starts.

And I think we could just give the player the code from the conversation?

Reply:
sounds good